common/dbc.py

```diff
@@ -15,7 +15,10 @@ def int_or_float(s):
 
 class dbc(object):
   def __init__(self, fn):
-    self.txt = open(fn).read().split("\n")
+
+    with open(fn, 'r') as datafile:
+      self.txt = map(str.split, datafile.read().split("\n"))
```

Comment:
This line doesn't do the same thing as the original. The original splits into lines, this splits more.